getting_started/developers_cli.adoc

```diff
@@ -140,13 +140,7 @@ ifdef::openshift-enterprise,openshift-origin,openshift-dedicated[]
 not have access, contact your cluster administrator.
 endif::[]
 ifdef::openshift-online[]
-- You must register for an account with the https://openshift.com[{product-title} (Next Gen) Developer Preview.]
-- Your project must be populated with the
-xref:../dev_guide/templates.adoc#using-the-instantapp-templates[Instant App
-templates] and xref:../using_images/s2i_images/index.adoc#using-images-s2i-images-index[builder images] that
-conform to the current xref:../getting_started/devpreview_faq.adoc#getting-started-devpreview-faq[set of
-considerations] for running in the {product-title} (Next Gen) Developer Preview
-offering.
+- You must register for an account with the https://openshift.com[{product-title} {product-version}.]
```

Comment:
Is it? https://openshift.com/online/starter ? Is that what this maps to?

I think the proper URL would be https://manage.openshift.com